app/javascript/util/listingUtil.ts

```diff
@@ -46,13 +46,13 @@ export const getFcfsSalesListingState = (listing: RailsSaleListing): ListingStat
     return ListingState.Closed
   }
 
-  if (new Date(listing.Application_Start_Date_Time) < new Date()) {
-    console.log("open")
+  if (dayjs(listing.Application_Start_Date_Time) < dayjs()) {
```

Comment:
why this change?

Reply:
1. We shouldn't log to the console
2. `dayjs` is what we use for _most_ date/time processing, for consistency I think we gradually move away from `Date` when possible

Reply:
gotcha